src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+// Add more methods here
+
+export const addTodo = (todo: Todo) => {
+  return client.post<Todo>('/todos', todo);
+};
+
+export const deleteTodo = (id: number) => {
+  return client.delete(`/todos/${id}`)
+    .catch(() => {
```

Comment:
It is better to handle errors in the component where the API is being called, so you can display specific error messages or handle it based on the component's requirements.